src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -102,10 +104,25 @@ internal class InProcLanguageServer : IAsyncDisposable
         /// capabilities.  The specification assures that the initialize request is sent only once.
         /// </summary>
         [JsonRpcMethod(Methods.InitializeName, UseSingleObjectParameterDeserialization = true)]
-        public Task<InitializeResult> InitializeAsync(InitializeParams initializeParams, CancellationToken cancellationToken)
+        public Task<InitializeResult> InitializeAsync(JToken initializeParams, CancellationToken cancellationToken)
         {
             Contract.ThrowIfTrue(_clientCapabilities != null, $"{nameof(InitializeAsync)} called multiple times");
-            _clientCapabilities = (VSClientCapabilities)initializeParams.Capabilities;
+
+            // Workaround: Allow LSP server to initialize on pre 16.9 preview 4 VS installations by handling the old tag support
+            // property type.  Once integration test machines update this can be removed.
```

Comment:
tracking issue for this please.

Reply:
looks like the machines are upgraded to P4 now, so I've removed this check